internal/cli/root.go

```diff
@@ -125,9 +131,8 @@ func NewRootCmd() (*cobra.Command, error) {
 
 	rootCmd.AddCommand(
 		CmdCompletion,
-		CmdSettings,
```

Comment:
Is `CmdSettings` gone ?

Reply:
:man_facepalming: fixed with a281f0e9b1193be5f8969521bcbbd309a094a520